src/SFA.DAS.EmployerIncentives.Api/Controllers/CollectionCalendarController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Api.Models;
+using SFA.DAS.EmployerIncentives.Application.Commands.CollectionsCalendar;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    public class CollectionCalendarController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<CollectionCalendarController> _logger;
+
+        public CollectionCalendarController(IMediator mediator, ILogger<CollectionCalendarController> logger)
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        [HttpPatch]
+        [Route("/collectionCalendar/period/active")]
```

Comment:
Shouldn't this be /collectionPeriods and the patch is just the CollectionPeriod data to be updated